polaris-gateway/Validators/ValidatorHelper.cs

```diff
@@ -1,16 +1,17 @@
 using FluentValidation;
 using Microsoft.AspNetCore.Http;
-using Microsoft.Azure.WebJobs.Extensions.Http;
-using Newtonsoft.Json;
+using System.Text.Json;
+using System.Threading.Tasks;
 
 namespace PolarisGateway.Validators
 {
     public class ValidatorHelper
     {
         public static async Task<ValidatableRequest<T>> GetJsonBody<T, V>(HttpRequest request) where V : AbstractValidator<T>, new()
         {
-            var requestJson = await request.ReadAsStringAsync();
-            var requestObject = JsonConvert.DeserializeObject<T>(requestJson);
+            request.Headers.Remove("Content-Type");
+            request.Headers.Append("Content-Type", "application/json");
```

Comment:
Why the validator modifies the request headers?

Reply:
Will raise another ticket to cover some general tidy up tasks to address minor comments such as this.